test/lit.cfg

```diff
@@ -1891,6 +1885,17 @@ if not kIsWindows:
 			"SIMCTL_CHILD_DYLD_LIBRARY_PATH='{0}' " # Simulator option
 			.format(all_stdlib_path, libdispatch_path)) + config.target_run
 
+subst_target_jit_prefix = ""
+if platform.system() == 'Darwin' and config.target_run:
```

Comment:
At one point you had a big comment about why this was needed; it would be nice to resurrect that.